graph-subscriptions-rs/src/subscription_tier.rs

```diff
@@ -26,6 +26,7 @@ impl SubscriptionTiers {
     pub fn tier_for_rate(&self, sub_rate: u128) -> SubscriptionTier {
         self.0
             .iter()
+            .rev()
```

Comment:
Oh that makes sense. I bumped the ECMAScript target in the frontend output to use `Array.prototype.findLast`.

https://github.com/edgeandnode/subgraph-studio/blob/d7bbdf76550aaa0d3c6a37c4601739aeb1047d13/packages/ui/src/billing-2-0/BillingPlanSelection/BillingPlanSelection.tsx#L73